train.py

```diff
@@ -481,12 +494,37 @@ def main(args):
 			onnxruntime.set_default_logger_severity(0)
 		(logits_, ) = onnxruntime_session.run(None, dict(x = waveform_input.cpu().numpy()))
 
-		assert torch.allclose(
+		wraper = models.OnnxWrapper(args.onnx)
+		wraper_logits = wraper(waveform_input)['logits'][0]
+
+		print('pytorch with wrapper max difference: ', (logits - wraper_logits).abs().max())
+
+		pytorch_with_onnx = torch.allclose(
 				logits.cpu(),
 				torch.from_numpy(logits_),
 				**{'rtol': 1e-01, 'atol': 1e-02} if args.fp16 else {'rtol': 1e-02, 'atol': 1e-03}
 		)
 
+		wrapper_with_onnx = torch.allclose(
```

Comment:
I guess all this code is not needed anymore

Reply:
done